easy-ui-react/src/DataGrid/DataGrid.stories.tsx

```diff
@@ -0,0 +1,229 @@
+import { action } from "@storybook/addon-actions";
+import { Meta, StoryObj } from "@storybook/react";
+import React, { Key } from "react";
+import { useAsyncList } from "react-stately";
+import { Menu } from "../Menu";
+import {
+  PlaceholderBox,
+  createNaiveSortingFunction,
+} from "../utilities/storybook";
+import { DataGrid } from "./DataGrid";
+import { DataGridProps } from "./types";
+
+type Story = StoryObj<typeof DataGrid>;
+
+const columns = [
+  { key: "name", name: "Name" },
+  { key: "email", name: "Email" },
+  { key: "permissions", name: "Permissions" },
+  { key: "status", name: "Status" },
+  { key: "lastActive", name: "Last Active" },
+];
+
+const rows = [
+  {
+    key: 1,
+    name: "Julie Smith",
+    email: "julie.smith@easypost.com",
```

Comment:
Consider using example.com instead in the (unlikely) event we get a Julie Smith or Sam Frost working here some day